### ISSUE TITLE
iOS Compatibility

### ISSUE THREAD

**Issue by thommcgrath** [ktekinay/M_Crypto#2]
The module looks close, but Encryptor still uses a lot of String methods. Any plans for full iOS/new framework compatibility?

**Comment by ktekinay** [comment]
No, not until Text handling on Windows/Linux is fixed (sped up). But if you want to tackle it, feel free.

**Comment by thommcgrath** [comment]
Well I don't need it yet, but I could in the future, so we'll see what the status is then. If I have to do it, I will.

**Comment by ktekinay** [comment]
My implementation might not be great for iOS anyway. To save on computing time, I cache values that reserve about a meg of RAM the first time you use AES, for example. This results in a huge speed boost but would not play well on an iOS device, I'd guess.
